secp256k1-zkp-dev/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+// secp256k1-zkp bindings
+// Written in 2019 by
+//   Jonas Nick
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the CC0 Public Domain Dedication
+// along with this software.
+// If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+//
+
+/// The functions in this module are copied from secp256k1 because they can only be used when
+/// compiled with the `rand` feature. But the other libraries need them as a dev-dependency for
+/// `cargo test` also when `rand` is not enabled.
```

Comment:
Is this https://github.com/rust-lang/cargo/issues/1796 ? Can you link to that issue in this comment?

Reply:
Ah, yes.